src/Pyz/Zed/Oms/OmsDependencyProvider.php

```diff
@@ -50,21 +50,8 @@ class OmsDependencyProvider extends SprykerOmsDependencyProvider
     public function provideBusinessLayerDependencies(Container $container): Container
     {
         $container = parent::provideBusinessLayerDependencies($container);
-        $container = $this->extendConditionPlugins($container);
-        $container->extend(self::COMMAND_PLUGINS, function (CommandCollectionInterface $commandCollection) {
-            $commandCollection->add(new SendOrderConfirmationPlugin(), 'Oms/SendOrderConfirmation');
-            $commandCollection->add(new SendOrderShippedPlugin(), 'Oms/SendOrderShipped');
-            $commandCollection->add(new StartReturnCommandPlugin(), 'Return/StartReturn');
-            $commandCollection->add(new GenerateOrderInvoiceCommandPlugin(), 'Invoice/Generate');
-            $commandCollection->add(new CreateMerchantOrdersCommandPlugin(), 'MerchantSalesOrder/CreateOrders');
-            $commandCollection->add(new CloseMerchantOrderItemCommandPlugin(), 'MerchantOms/CloseOrderItem');
-            $commandCollection->add(new SendEventPaymentConfirmationPendingPlugin(), 'Payment/SendEventPaymentConfirmationPending');
-            $commandCollection->add(new SendEventPaymentRefundPendingPlugin(), 'Payment/SendEventPaymentRefundPending');
-            $commandCollection->add(new SendEventPaymentCancelReservationPendingPlugin(), 'Payment/SendEventPaymentCancelReservationPending');
-
-            return $commandCollection;
-        });
         $container = $this->extendCommandPlugins($container);
+        $container = $this->extendConditionPlugins($container);
```

Comment:
Just fix the vertical ordering of functions to be the same order in calling and its place in class
@ilantipov

Reply:
Change also the order of function in the class itself

```
public function provideBusinessLayerDependencies(Container $container): Container
    {
        $container = parent::provideBusinessLayerDependencies($container);
        $container = $this->extendConditionPlugins($container);
        $container = $this->extendCommandPlugins($container);
        return $container;
    }
```
``` 
protected function extendConditionPlugins(Container $container): Container
    {
        //function body
    }
```
``` 
protected function extendCommandPlugins(Container $container): Container
    {
        //function body
    }
```

Reply:
I think it's an unnecessary change, because it is not related neither to issue solved nor to code touched

Reply:
It's just a code smell, you have extracted `commandPlugins` so you have to make it right from vertical ordering so when anybody reads the code he does not need to jump to the end of the file to read a function that has been calling on top of your class/file

Reply:
Please try to avoid such not necessary changes, I agreed that it would be a good practice but for now, it will add pain while merging b2b to b2b-marketplace demo shops.